services/ui-src/src/components/pages/Dashboard/DashboardPage.tsx

```diff
@@ -86,16 +92,21 @@ export const DashboardPage = ({ reportType }: Props) => {
     let newReportsToDisplay = reportsByState;
     if (!userIsAdmin) {
       newReportsToDisplay = reportsByState?.filter(
-        (report: ReportShape) => !report?.archived
+        (report: DashboardReportShape) => !report?.archived
       );
     }
     setReportsToDisplay(newReportsToDisplay);
   }, [reportsByState]);
 
-  const enterSelectedReport = async (report: ReportShape) => {
+  const enterSelectedReport = async (report: DashboardReportShape) => {
```

Comment:
Thought: I think we should wrap this in a try/catch block in case the report cannot be loaded for whatever reason and add some error handling here that displays an error message if it can't be loaded and entered. Might have to be a follow-up item.

Reply:
Agree - [Heres a ticket to add this in as a separate PR. ](https://qmacbis.atlassian.net/browse/MDCT-2143)